jmclient/jmclient/wallet.py

```diff
@@ -73,18 +89,32 @@ def estimate_tx_fee(ins, outs, txtype='p2pkh', outtype=None, extra_bytes=0):
             btc.fee_per_kb_to_str(fee_per_kb) +
             " greater than absurd value " +
             btc.fee_per_kb_to_str(absurd_fee) + ", quitting.")
-    if txtype in ['p2pkh', 'p2shMofN']:
-        tx_estimated_bytes = btc.estimate_tx_size(ins, outs, txtype, outtype) + extra_bytes
+
+    # See docstring for explanation:
+    if isinstance(txtype, str):
+        ins = [txtype]* ins
```

Comment:
nit
```suggestion
        ins = [txtype] * ins
```

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1421/commits/357b61165551ac1e096527bc9f372c2c03377404